frontend/src/reviews/ReviewComments.tsx

```diff
@@ -53,34 +47,51 @@ export default function ReviewComments({ release, accessRequest, isEdit }: Revie
     modelId,
     ...semverOrAccessRequestIdObject,
   })
+  const { responses, isResponsesLoading, isResponsesError, mutateResponses } = useGetResponses([
+    release ? release._id : accessRequest._id,
+    ...reviews.map((review) => review._id),
+  ])
 
   const hasResponseOrComment = useMemo(() => {
-    const hasReviewResponse = !!reviews.find((review) => review.responses.length > 0)
-    const hasComment = release ? release.comments.length > 0 : accessRequest.comments.length > 0
+    const hasReviewResponse = responses.filter((response) => response.kind === ResponseKind.Review).length > 0
```

Comment:
To avoid having to iterate over the entire array you could do:
```
const hasReviewResponse = !!responses.find((response) => response.kind === ResponseKind.Review)
```